games-simulation/openttd/openttd-1.11.2.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="OpenTTD is a clone of Transport Tycoon Deluxe"
+HOMEPAGE="https://www.openttd.org/"
+SRC_URI="https://cdn.openttd.org/openttd-releases/${PV}/${P}-source.tar.xz"
+
+KEYWORDS="~amd64 ~ppc64 ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="allegro cpu_flags_x86_sse dedicated +fluidsynth icu +lzma lzo +openmedia +png +sdl timidity +truetype zlib"
+REQUIRED_USE="!dedicated? ( || ( allegro sdl ) )"
+
+RESTRICT="test" # needs a graphics set in order to test
+
+RDEPEND="
+	!dedicated? (
+		allegro? ( media-libs/allegro:5 )
+		fluidsynth? ( media-sound/fluidsynth )
+		icu? (
+			dev-libs/icu-layoutex
+			dev-libs/icu-le-hb
+			>=dev-libs/icu-58.1:=
+		)
+		sdl? ( media-libs/libsdl2[sound,video] )
+		truetype? (
+			media-libs/fontconfig
+			media-libs/freetype:2
+			sys-libs/zlib:=
+		)
+	)
+	lzma? ( app-arch/xz-utils )
+	lzo? ( dev-libs/lzo:2 )
+	png? ( media-libs/libpng:0=	)
+	zlib? ( sys-libs/zlib:= )"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+PDEPEND="
+	!dedicated? (
+		openmedia? (
+			>=games-misc/openmsx-0.4.0
+			>=games-misc/opensfx-1.0.1
+		)
+	)
+	openmedia? ( >=games-misc/opengfx-0.6.1 )
+	timidity? ( media-sound/timidity++ )"
+
+DOCS=( docs/directory_structure.md )
+
+src_prepare() {
+	sed -i -e "/COMMAND gzip -9 -n -f/d" -e "s/.gz//" \
+		cmake/InstallAndPackage.cmake || die "sed failed"
```

Comment:
`sed` will fail silently. Use a patch instead to detect any upstream changes on a future version bump.